### PR TITLE
Ci/monorepo docker build

### PR DESCRIPTION
**Pull request by jonathangoulding**
**Removes Lerna**
> You need to remove all node modules and package locks if present locally (in the packages) 
> Useful command - `find . -name "node_modules" -exec rm -rf '{}' +; find . -name "package-lock.json" -exec rm -rf '{}' +; ` - if you sue this command do a pull and get the root package lock back into your local env.

**Uses npm workspaces** 
To install all node modules (now most will be in the root node modules) run npm i or ci (packages are now hosited to the root node modules)

No package versions are used and the versioning is now in the root package.json (This means no more publishing to npm and the containers having access to the whole monorep)

This also means no local conflicts with package versions and pulling from remote etc and out of sync version. 

**Version**
After removing lerna we need a simple versioning process until we move to a more refined solution. the docs will be updated

**Docker**
Docker stacks have been removed and replaced with docker composes. You will need to reseed the databases as the new volume will be wls_postgres

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
+    "docker:stop-services": "docker-compose -f docker/services.yml down",
+    "docker:start-cloud": "docker-compose -f docker/cloud-services.yml up -d",
+    "docker:stop-cloud": "docker-compose -f docker/cloud-services.yml down",
+    "docker:start-proxy": "docker-compose -f docker/nginx/nginx-service.yml up -d ",
+    "docker:stop-proxy": "docker-compose -f docker/nginx/nginx-service.yml down",
+    "prerelease:patch": "npm version prepatch --preid=alpha",
+    "prerelease:minor": "npm version preminor --preid=alpha",
+    "prerelease:major": "npm version premajor --preid=alpha",
+    "postversion": "git push && git push --tags"
+  },
```
Comment (richard-mohammed-def) [inline package.json:31]: Would really help to have readme updated to reflect these script changes and ideally write down the new build and deploy process

File: packages/connectors-lib/README.md [hunk 2]
```diff
@@ -4,3 +4,4 @@
 
 Common connectors library for wildlife licencing
 
+A change
```
Comment (richard-mohammed-def) [inline packages/connectors-lib/README.md:7]: yep. A change

File: packages/web-service/src/server.js [hunk 3]
```diff
@@ -91,9 +91,9 @@ const init = async server => {
     isCached: process.env.NODE_ENV !== 'development',
 
     path: [
-      path.join(__dirname, 'node_modules', GOVUK_FRONTEND),
-      path.join(__dirname, 'node_modules', GOVUK_FRONTEND, 'govuk'),
-      path.join(__dirname, 'node_modules', GOVUK_FRONTEND, 'govuk', 'components'),
+      path.join(__dirname, '/../../node_modules/', GOVUK_FRONTEND),
+      path.join(__dirname, '/../../node_modules/', GOVUK_FRONTEND, 'govuk'),
+      path.join(__dirname, '/../../node_modules/', GOVUK_FRONTEND, 'govuk', 'components'),
```
Comment (richard-mohammed-def) [inline packages/web-service/src/server.js:96]: Create a constant for node_modules path?

File: package.json (hunk 1 above)
Comment (jonathangoulding) [reply package.json:31]: Readme updated for the docker compose work. 

I can update the release reamdme here - https://github.com/DEFRA/wildlife-licencing-team

Once we are happy.

File: packages/connectors-lib/README.md (hunk 2 above)
Comment (jonathangoulding) [reply packages/connectors-lib/README.md:7]: good spot.

**Review by StuAA78 — CHANGES_REQUESTED** [review]
Just a few small tweaks

**Review by StuAA78 — APPROVED** [review]
Great stuff 👍

**Pull request merged**